src/jobs/keys-update/test/update-cases.spec.ts

```diff
@@ -0,0 +1,168 @@
+import { LOGGER_PROVIDER } from '@lido-nestjs/logger';
+import { Test, TestingModule } from '@nestjs/testing';
+import { ExecutionProviderService } from 'common/execution-provider';
+import { StakingRouterService } from 'staking-router-modules/staking-router.service';
+import { ElMetaStorageService } from 'storage/el-meta.storage';
+import { SRModuleStorageService } from 'storage/sr-module.storage';
+import { stakingModuleFixture, stakingModuleFixtures } from '../keys-update.fixtures';
```

Comment:
we could move fixtures to test too

Reply:
Could you explain?

Reply:
i mean to put keys-update.fixtures in test  folder near update-cases.spec.ts

Reply:
done